### PR TITLE
Randomize test file

### PR DESCRIPTION
**Pull request by Ostrokrzew**
Change test file created by function to randomly generated string with custom lenght.
Remane function to more appropriate.

Signed-off-by: Slawomir Jankowski <slawomir.jankowski@intel.com>

### CODE REVIEW

File: test_tools/fs_utils.py [hunk 1]
```diff
@@ -280,9 +282,11 @@ def parse_ls_output(ls_output, dir_path=''):
...
 
 
-def create_test_file(target_file_path, file_content="Test content"):
+def create_random_test_file(target_file_path: str,
+                            file_size: Size = Size(1, Unit.MebiByte)):
     from test_utils.filesystem.file import File
     file = File.create_file(target_file_path)
-    file.write(file_content)
+    file.write("".join(random.choice(string.digits + string.ascii_letters + string.punctuation)
+                       for _ in range(file_size.value)))
```
Comment (robertbaldyga) [inline test_tools/fs_utils.py]: `file.write()` transfers data from controller to DUT via ssh and it should never be used to write files bigger that few megabytes. Some test cases may want to create files that are much bigger, thus it would be better to have method that generates file entirely on DUTs side.

File: test_tools/fs_utils.py [hunk 2]
```diff
@@ -65,9 +67,9 @@ def copy(source: str,
          recursive: bool = False,
          dereference: bool = False):
     cmd = f"cp{' --force' if force else ''}" \
-        f"{' --recursive' if recursive else ''}" \
-        f"{' --dereference' if dereference else ''} " \
-        f"{source} {destination}"
+          f"{' --recursive' if recursive else ''}" \
+          f"{' --dereference' if dereference else ''} " \
+          f"{source} {destination}"
     return TestRun.executor.run_expect_success(cmd)
```
Comment (robertbaldyga) [inline test_tools/fs_utils.py]: Coding style improvements should be separated into another patch.

File: test_tools/fs_utils.py (hunk 1 above)
Comment (Ostrokrzew) [reply test_tools/fs_utils.py]: I modified it to use DD. Is it better solution?

Comment (robertbaldyga) [reply test_tools/fs_utils.py]: It's better but not perfect. My biggest concern is `block_size(Size(1, Unit.Byte))`. 
Consider following results:
```
# dd if=/dev/zero of=/dev/nvme0n1 bs=131072 count=800
800+0 records in
800+0 records out 
104857600 bytes (105 MB, 100 MiB) copied, 0.136058 s, 771 MB/s

# dd if=/dev/zero of=/dev/nvme0n1 bs=4096 count=25600
25600+0 records in
25600+0 records out 
104857600 bytes (105 MB, 100 MiB) copied, 0.175051 s, 599 MB/s

# dd if=/dev/zero of=/dev/nvme0n1 bs=512 count=204800
204800+0 records in
204800+0 records out 
104857600 bytes (105 MB, 100 MiB) copied, 0.851134 s, 123 MB/s

# dd if=/dev/zero of=/dev/nvme0n1 bs=1 count=104857600
104857600+0 records in
104857600+0 records out 
104857600 bytes (105 MB, 100 MiB) copied, 136.207 s, 770 kB/s
```
In each case we write 100MiB with different block sizes. Difference between fastest and slowest case is ~100000%.  We **really** want to have bs=512 or bigger.

Comment (Ostrokrzew) [reply test_tools/fs_utils.py]: Gave 1 MiB

Comment (Ostrokrzew) [reply test_tools/fs_utils.py]: Gave 1 MiB.

File: test_tools/fs_utils.py [hunk 3]
```diff
@@ -280,9 +281,16 @@ def parse_ls_output(ls_output, dir_path=''):
...
+                            file_size: Size = Size(1, Unit.MebiByte),
+                            random: bool = True):
     from test_utils.filesystem.file import File
+    bs = Size(1, Unit.MebiByte)
     file = File.create_file(target_file_path)
-    file.write(file_content)
+    dd = Dd().output(target_file_path) \
+             .input("/dev/urandom" if random else "/dev/zero") \
+             .block_size(bs) \
+             .count(int(file_size.value / bs.value))
```
Comment (robertbaldyga) [inline test_tools/fs_utils.py]: So now if we try to create file smaller than 1MiB, the count will be 0.

Comment (Ostrokrzew) [reply test_tools/fs_utils.py]: Changed to 512 KiB and add check for value under 1

File: test_tools/fs_utils.py [hunk 4]
```diff
@@ -280,9 +281,17 @@ def parse_ls_output(ls_output, dir_path=''):
     return fs_items
 
 
-def create_test_file(target_file_path, file_content="Test content"):
+def create_random_test_file(target_file_path: str,
+                            file_size: Size = Size(1, Unit.MebiByte),
+                            random: bool = True):
     from test_utils.filesystem.file import File
+    bs = Size(512, Unit.KibiByte)
+    cnt = int(file_size.value / bs.value)
```
Comment (robertbaldyga) [inline test_tools/fs_utils.py]: It would be better to have entire `cnt` calculation procedure in one place:
```python
cnt = math.ceil(file_size.value / bs.value)
```
